src/DssSpell.sol

```diff
@@ -42,66 +38,94 @@ contract DssSpellAction is DssAction {
-        DssExecLib.setIlkStabilityFee("ETH-B", THREE_PT_TWO_FIVE_PCT_RATE, true);
-
-        // Increase ETH-C Stability Fee by 0.25% from 0.75% to 1%.
-        DssExecLib.setIlkStabilityFee("ETH-C", ONE_PCT_RATE, true);
-
-        // Increase WSTETH-A Stability Fee by 0.25% from 1.5% to 1.75%.
-        DssExecLib.setIlkStabilityFee("WSTETH-A", ONE_PT_SEVEN_FIVE_PCT_RATE, true);
-
-        // Increase WSTETH-B Stability Fee by 0.25% from 0.75% to 1%.
-        DssExecLib.setIlkStabilityFee("WSTETH-B", ONE_PCT_RATE, true);
-
-        // Increase RETH-A Stability Fee by 0.25% from 0.5% to 0.75%.
-        DssExecLib.setIlkStabilityFee("RETH-A", ZERO_PT_SEVEN_FIVE_PCT_RATE, true);
-
-        // Increase CRVV1ETHSTETH-A Stability Fee by 0.25% from 1.5% to 1.75%.
-        // NOTE: ignore in goerli because CRVV1ETHSTETH-A doesn't exist there.
-        // DssExecLib.setIlkStabilityFee("CRVV1ETHSTETH-A", ONE_PT_SEVEN_FIVE_PCT_RATE, true);
-
-
-        // Increase the WSTETH-A gap by 15 million DAI from 15 million DAI to 30 million DAI.
-        // Increase the WSTETH-A ttl by 21,600 seconds from 21,600 seconds to 43,200 seconds
-        DssExecLib.setIlkAutoLineParameters("WSTETH-A", 500 * MILLION, 30 * MILLION, 12 hours);
-
-        // Increase the WSTETH-B gap by 15 million DAI from 15 million DAI to 30 million DAI.
-        // Increase the WSTETH-B ttl by 28,800 seconds from 28,800 seconds to 57,600 seconds.
-        DssExecLib.setIlkAutoLineParameters("WSTETH-B", 500 * MILLION, 30 * MILLION, 16 hours);
-
-        // Reduce the WBTC-A gap by 10 million DAI from 20 million DAI to 10 million DAI.
-        DssExecLib.setIlkAutoLineParameters("WBTC-A", 500 * MILLION, 10 * MILLION, 24 hours);
-
-        // Reduce the WBTC-B gap by 5 million DAI from 10 million DAI to 5 million DAI.
-        DssExecLib.setIlkAutoLineParameters("WBTC-B", 250 * MILLION, 5 * MILLION, 24 hours);
-
-        // Reduce the WBTC-C gap by 10 million DAI from 20 million DAI to 10 million DAI.
-        DssExecLib.setIlkAutoLineParameters("WBTC-C", 500 * MILLION, 10 * MILLION, 24 hours);
+        // --------- Collateral Offboardings ---------
+        // Poll: https://vote.makerdao.com/polling/QmPwHhLT#poll-detail
+        // Forum: https://forum.makerdao.com/t/decentralized-collateral-scope-parameter-changes-1-april-2023/20302
+
+        // Set Liquidation Penalty (chop) to 0%.
+        DssExecLib.setIlkLiquidationPenalty("YFI-A", 0);
+        // Set Flat Kick Incentive (tip) to 0.
+        DssExecLib.setKeeperIncentiveFlatRate("YFI-A", 0);
+        // Set Proportional Kick Incentive (chip) to 0.
+        DssExecLib.setKeeperIncentivePercent("YFI-A", 0);
+        // Set Liquidation Ratio (mat) to 10,000%.
+        // We are using low level methods because DssExecLib only allows setting `mat < 1000%`: https://github.com/makerdao/dss-exec-lib/blob/69b658f35d8618272cd139dfc18c5713caf6b96b/src/DssExecLib.sol#L717
+        DssExecLib.setValue(MCD_SPOT, "YFI-A", "mat", 100 * RAY);
```

Comment:
The sheet says set LR (mat) to `10_000%`.
While there is nothing wrong with this statement, it requires some implicit knowledge:

> In MakerDAO Spot contract, `mat` (liquidation ratio) has `10**27` precision.
> 100% Liquidation Ratio = 1 RAY
> 10,000% Liquidation Ration = 100 RAY

This becomes obvious after you are familiar with the system, but can be helpful to properly document it for someone having contact with spells for the first time.

There's no need to repeat the comment everywhere though, just in the first occurrence should be fine.

Reply:
I agree that it's confusing on the first sight, but actually quite understandable that we're not using % in the code, but rather plain multiplication value. Moreover, as soon as this value is tested using different notation, I decided it's fine to keep it as is to avoid requesting another review iteration to unblock the deployment